docs/operation/operation.md

```diff
@@ -921,6 +921,12 @@ The parameters of `-passive-health-check` option are:
 + `max-drop-probabilty=<float more than/equal to 0 and less than/equal to 1>` - the maximum possible probability of unhealthy endpoint being not considered
 while choosing the endpoint for the given request
 
+### PHC Metrics
```

Comment:
minor: as this is already subsection of "Passive health check (*experimental*)", "PHC" here could be omitted. Just "Metrics", and that's it.